src/pps/pps.proto

```diff
@@ -158,13 +158,12 @@ message Datum {
 }
 
 enum DatumState {
-  DATUM_STATE_UNKNOWN = 0;
+  DATUM_STATE_UNKNOWN = 0; // or not part of a job
```

Comment:
Nit, but this could just be unknown. The datum state part is implicit in the type.